content/pages/disability-benefits/after-you-apply.md

```diff
@@ -73,15 +73,15 @@ If we need more evidence during the review, your claim may return to this step m
 
 </li>
 
-<li class="step four">
+<li class="process-step list-four">
  
 #### Preparation for Notification
 
 We’ll get your entire claim decision packet ready to be mailed.
 
 </li>
 
-<li class="step last five">
+<li class="process-step last list-five">
```

Comment:
@tiffanybbrown-va a few of these lists still have the `.last` class on the last list item - that can be removed, right?